### DISCUSSION TITLE
Convert from ECEF to ECI

### DISCUSSION THREAD

**Discussion by natczicro** [skyfielders/python-skyfield#925] [Q&A]
Hello I'm having a bit of trouble figuring out how to use Skyfield to convert between the Earth Centred Earth Fixed reference frame and the Earth Centred Inertial frame for some satellite positions and velocities.  I have the coordinates in the ITRF or ECEF frame and so just need to apply the transformation in order to get them in the ECI frame.

I'm assuming I can use the 'itrs' reference frame and the SkyCoord object for ECEF

ITRS_position = SkyCoord(x=satellitePositionX, y=satellitePositionY, z=satellitePositionZ, frame='itrs', representation_type='cartesian',unit='m')

But it's not immediately clear if any of the other reference frames would correspond to the ECI reference frame.

I figure it should be fairly straightforward and I'm just overlooking something.

**Comment by brandon-rhodes** [comment]
@natczicro — My guess is that you will get a better response to your question over on an AstroPy discussion forum, as Skyfield itself does not have a class named `SkyCoord`, which is an AstroPy feature. Did you perhaps intend to ask your question over there, and had this forum open by mistake?

**Comment by natczicro** [comment]
Oops you're right!  I was going back and forth between trying the different libraries and lost track of which one I was using

I did manage to find another library well suited for what I wanted in any event (https://github.com/geospace-code/pymap3d)

**Reply by brandon-rhodes** [reply]
Thanks for sharing a link to another useful library, and thanks for coming back to close the Discussion.
